scripts/eda_kube.sh

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env bash
+
```

Comment:
The `nounset` option is actually needed, because it not only validates externally passed parameters, but also any variables (including local ones) used in the script. This is extremely useful in case of typos in variable names for instance, otherwise it leads to sometimes unexpected side effects.

Reply:
Done